pkgs/development/tools/apksigcopier/default.nix

```diff
@@ -42,12 +42,12 @@ python3.pkgs.buildPythonApplication rec {
   '';
 
   meta = with lib; {
-    description = "Copy/extract/patch apk signatures & compare apks";
+    description = "Copy/extract/patch android apk signatures & compare apks";
     longDescription = ''
-      apksigcopier is a tool for copying APK signatures from a signed APK
-      to an unsigned one (in order to verify reproducible builds).  It can
-      also be used to compare two APKs with different signatures.  Its
-      command-line tool offers four operations:
+      apksigcopier is a tool for copying android APK signatures from a
+      signed APK to an unsigned one (in order to verify reproducible
+      builds).  It can also be used to compare two APKs with different
+      signatures.  Its command-line tool offers four operations:
```

Comment:
```suggestion
      builds). It can also be used to compare two APKs with different
      signatures. Its command-line tool offers four operations:
```

Reply:
I can change it if you really want me to, but I prefer to use two spaces after a full stop.

(of course it won't matter for e.g. HTML, it is common in e.g. terminal-based environments where it does render as 2 spaces)

Reply:
The two spaces are not rendered at search.nixos.org anyway together with the reflowing. We might as well remove that to make it easier to read.

Reply:
I guess there's no good way to show the long description w/ `nix-env` :(